lib/config-win32.h

```diff
@@ -502,9 +502,20 @@
 #define _CRT_NONSTDC_NO_DEPRECATE 1
 #endif
 
+/* mingw-w64 or mingw */
+#ifdef __MINGW32__
+#  include <_mingw.h>
+#  ifdef __MINGW64_VERSION_MAJOR  /* mingw-w64 */
+#    ifndef _USE_32BIT_TIME_T
+#      define SIZEOF_TIME_T 8
+#    else
+#      define SIZEOF_TIME_T 4
+#    endif
+#  endif
+#  define SIZEOF_TIME_T 8
```

Comment:
but how do you know this? my mingw original has a (similar) check as mingw-w64 as I noted in my post.

Reply:
I didn't, I left it there by mistake. Thanks for pointing out!